cli/cmd/capture/capture.go

```diff
@@ -9,7 +9,10 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-var name string
+var opts = struct {
```

Comment:
Maybe we can borrow what kubectl cli has been achieved here? Like https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/create/create_cronjob.go

Reply:
^^ it's for this change, we can improve later.
Thanks for fixing missing namespace flag.

Reply:
That's the direction I am moving with this change, but it is just the first small step